src/Locator/FileNotFound.php

```diff
@@ -63,8 +75,27 @@ public static function multipleFilesDoNotExist(string $path, array $files): self
         );
     }
 
-    public static function filesNotFound(): self
+    /**
+     * @param string[] $files
+     * @param string[] $roots
+     */
+    public static function createForFiles(array $files, array $roots): self
```

Comment:
Perhaps `fromFilenames()` is a better choice, along with `$fileNames` as parameter name?